lib/src/ui/pages/location/location_info_pin.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:allthenews/generated/l10n.dart';
+import 'package:allthenews/src/domain/location/location.dart';
+import 'package:flutter/material.dart';
+
+class _Constants {
+  static const widthScreenRatio = 0.70;
+  static const height = 80.0;
+  static const padding = 20.0;
+  static const radius = 50.0;
+}
+
+class LocationInfoPin extends StatelessWidget {
```

Comment:
zmieniłbym nazwę na coś w stylu `LocationInfoLabel`, pin  kojarzy mi się ze znacznikiem na mapie, jakby wyszukać `pin` w googlach to większość wyników to coś takiego 
![image](https://user-images.githubusercontent.com/24854180/104091594-a30fa300-527e-11eb-86fb-c52959791b50.png)